splice_ck/src/main/java/com/splicemachine/ck/command/RPutCommand.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2012 - 2020 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.splicemachine.ck.command;
+
+import com.splicemachine.ck.HBaseInspector;
+import com.splicemachine.ck.Utils;
+import com.splicemachine.ck.encoder.RPutConfigBuilder;
+import org.apache.commons.lang3.StringUtils;
+import picocli.CommandLine;
+
+import java.util.concurrent.Callable;
+
+
+@CommandLine.Command(name = "rput",
+        description = "modify HBase row",
```

Comment:
I would add quite the disclaimer here, informing that doing so may compromise data integrity, break our MVCC model, etc.

Reply:
Good idea, I will do that.